test/verify/check-multi-machine

```diff
@@ -381,10 +381,10 @@ class TestMultiMachine(MachineCase):
 
         # Connect to bad machine
         b.set_val("#login-password-input", "bad-password")
-        b.click('#option-group span')
```

Comment:
Selecting things by referring them to `span` or `div` is asking for problems.

It's better if we're selecting by an element with semantic meaning... or in most cases, it's even better if we're selecting ID or classname.